registry/src/registry_helper.rs

```diff
@@ -72,7 +72,10 @@ impl RegistryHelper {
             Ok(Registry {
                 key_path: self.config.key_path.clone(),
                 value_name: Some(value_name.clone()),
-                value_data: Some(convert_reg_value(&value)?),
+                value_data: match value {
+                    Data::None => None,
```

Comment:
Is this `match` statement necessary? Looks like `convert_reg_value()` has been updated to handle the `Data::None` case.

Reply:
Not exactly, but you bring up a good point, `convert_reg_value()` should return an `Option` instead and this match wouldn't be needed.  I'll change that.